### PR TITLE
Create datatype_tabular.md

### PR DESCRIPTION
**Pull request by Beachyesh**
Proposing this file be added to galaxy FAQs. Please suggest any changes that need to be made, including adding any contributor names.

### CODE REVIEW

**Comment by Beachyesh** [comment]
@hexylena How can I fix this failed check

**Comment by hexylena** [comment]
hi @Beachyesh for fixing this error (I guess you deleted the comment), please add yourself to the CONTRIBUTORS.yaml file.

File: faqs/galaxy/datatype_tabular.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Datatype Tabular
+area: Getting inputs right    # FAQs will be grouped by these areas on the FAQ page
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+contributors: [beachyesh] 
+---
+
+[Back to Support Hub](https://galaxyproject.org/support/)
```
Comment (hexylena) [inline faqs/galaxy/datatype_tabular.md:9]: please do not include this.

File: faqs/galaxy/datatype_tabular.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Related Topics 
+- [Fastq Manipulation and Quality Control](https://galaxyproject.org/tutorials/ngs/#fastq-manipulation-and-quality-control)  
+- [How to format fastq data for tools that require .fastqsanger format?](https://galaxyproject.org/support/fastqsanger/)
+- [Understanding compressed fastq data (fastq.gz)](https://galaxyproject.org/support/compressed-fastq/)
+- [Common datatypes explained](https://galaxyproject.org/learn/datatypes/)
+- [Input datatype misassignment and errors](https://galaxyproject.org/support/tool-error/)
```
Comment (hexylena) [inline faqs/galaxy/datatype_tabular.md:16]: this is interesting but I'm not sure we should include this especially when e.g. `Understanding compressed fastq data (fastq.gz)` will probably become a faq sometday.

I would suggest removing this

File: faqs/galaxy/datatype_tabular.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: Datatype Tabular
+area: Getting inputs right    # FAQs will be grouped by these areas on the FAQ page
```
Comment (shiltemann) [inline faqs/galaxy/datatype_tabular.md:3]: please try to align with existing areas, this sounds more like a title

File: faqs/galaxy/datatype_tabular.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+
```
Comment (shiltemann) [inline faqs/galaxy/datatype_tabular.md:6]: please remove this empty line, it may break the rendering

File: faqs/galaxy/datatype_tabular.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+---
+ 
+### Help for Tabular Datasets
```
Comment (shiltemann) [inline faqs/galaxy/datatype_tabular.md:9]: you don't need a title here, the title defined in the metadata above will be placed at the top of the FAQ

**Comment by Beachyesh** [comment]
@shiltemann Got it, I'll make all the necessary changes. Thanks!

**Comment by Beachyesh** [comment]
@hexylena @shiltemann I updated the file to make it shorter and more aligned with how the rest of the faqs have been written. Let me know about further changes. Thank you for your input! :)

File: faqs/galaxy/datatype_tabular.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: Datatype Tabular
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:2]: Could we have a title here that indicates what this FAQ is trying to address?

File: faqs/galaxy/datatype_tabular.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabluar datatype is human readable and has tabs seperating data columns.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:9]:
```suggestion
- A Tabular datatype is human readable and has tabs separating data columns.
```

File: faqs/galaxy/datatype_tabular.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  - Note: tabular data is different from comma seperated data (.csv)
+- Common tabular datatypes are .bed, .gtf, .interval, or .txt.
+- The datatype metadata attribute can be directly reassigned to tabular format data.
+- Click the {% icon galaxy-pencil %} icon to reach the **_Edit Attributes_** form.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:13]: I think that this is where the FAQ starts and we could have the numbered steps

File: faqs/galaxy/datatype_tabular.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+  - Note: tabular data is different from comma seperated data (.csv)
+- Common tabular datatypes are .bed, .gtf, .interval, or .txt.
+- The datatype metadata attribute can be directly reassigned to tabular format data.
+- Click the {% icon galaxy-pencil %} icon to reach the **_Edit Attributes_** form. 
+  - Change the datatype (3rd tab) and save, then label columns (1st tab) and save.  Metadata will assign, then the dataset can be used.
+- If the required input is a BED or Interval datatype, adjusting (.tab → .bed, .tab → .interval) maybe possible using a combination of **_Text Manipulation_** tools, to create a dataset that matches the BED or Interval datatype specifications.
+
+#### Tips
+- Some tools require that BED format is followed, even if the datatype Interval (with less strict column ordering) is accepted on the tool form.
+- These tools will fail if run with malformed BED datasets or non-specific column assignments.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:19]: do you think that these tips could be included within the previous bullet point?

File: faqs/galaxy/datatype_tabular.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+**Solution**: Reorganize the data to be in BED format and rerun.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:21]: is this also a step in the FAQ?

File: faqs/galaxy/datatype_tabular.md (hunk 8 above)
Comment (shiltemann) [inline faqs/galaxy/datatype_tabular.md:13]: It is best if we also write out the icon description in text, followed by the icon itself, for increased accessibility (e.g. for screen readers)

```suggestion
- Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
```

File: faqs/galaxy/datatype_tabular.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+layout: faq          
+---
+ 
+
+- A Tabluar datatype is human readable and has tabs seperating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+- Common tabular datatypes are .bed, .gtf, .interval, or .txt.
+- The datatype metadata attribute can be directly reassigned to tabular format data.
+- Click the {% icon galaxy-pencil %} icon to reach the **_Edit Attributes_** form. 
+  - Change the datatype (3rd tab) and save, then label columns (1st tab) and save.  Metadata will assign, then the dataset can be used.
```
Comment (shiltemann) [inline faqs/galaxy/datatype_tabular.md:14]:
```suggestion
  - Change the datatype (3rd tab) and save, then label columns (1st tab) and save.  Metadata will be assigned, then the dataset can be used.
```

**Review by shiltemann — COMMENTED** [review]
Thanks @Beachyesh!

**Comment by Beachyesh** [comment]
@beatrizserrano I have updated the file.  Added a title, numbered the steps, and removed the tip to add a new bullet point. 
@shiltemann I have made the suggested commits as well. 
Always open to further improvement. Thanks to both of you for helping me out ❤️

File: faqs/galaxy/datatype_tabular.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
+2. The datatype metadata attribute can be directly reassigned to tabular format data.
+3. Click the {% icon galaxy-pencil %} ✏️ icon to reach the **_Edit Attributes_** form.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:13]: This `{% icon galaxy-pencil %}` translates into ✏️, so there's no need to add ✏️

File: faqs/galaxy/datatype_tabular.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
+2. The datatype metadata attribute can be directly reassigned to tabular format data.
+3. Click the {% icon galaxy-pencil %} ✏️ icon to reach the **_Edit Attributes_** form. 
+   - Change the datatype (3rd tab) and save
+   - label columns (1st tab) and save
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:15]:
```suggestion
   - Label columns (1st tab) and save
```

File: faqs/galaxy/datatype_tabular.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+   -  Metadata will be assigned, then the dataset can be used.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:16]:
```suggestion
   - Metadata will be assigned, then the dataset can be used.
```

File: faqs/galaxy/datatype_tabular.md [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
+2. The datatype metadata attribute can be directly reassigned to tabular format data.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:12]: I have a hard time understanding this sentence, could you rephrase it?

File: faqs/galaxy/datatype_tabular.md (hunk 12 above)
Comment (shiltemann) [inline faqs/galaxy/datatype_tabular.md:13]: We try to always also include the text description of the icon, followed by the icon itself, for accessibility reasons

```suggestion
3. Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
```

File: faqs/galaxy/datatype_tabular.md [hunk 16]
```diff
@@ -0,0 +1,20 @@
+---
+title: Indentifying and Formatting Tabular Dataset
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:2]:
```suggestion
title: Identifying and formatting Tabular Datasets
```

File: faqs/galaxy/datatype_tabular.md [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+title: Indentifying and Formatting Tabular Dataset 
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+layout: faq          
+---
+ 
+
+- A Tabular datatype is human readable and has tabs separating data columns.
+  - Note: tabular data is different from comma seperated data (.csv)
+1. Common tabular datatypes are `.bed`, `.gtf`, `.interval`, or `.txt`.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:11]: I think that this is a clarification, but not a proper step, same with step 2

File: faqs/galaxy/datatype_tabular.md [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+2. The datatype metadata attribute can be directly reassigned to tabular format data.
+3. Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
+   - Change the datatype (3rd tab) and save
+   - Label columns (1st tab) and save
+   - Metadata will be assigned, then the dataset can be used.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:16]: these 3 could maybe be converted into steps

File: faqs/galaxy/datatype_tabular.md (hunk 17 above)
Comment (Beachyesh) [reply faqs/galaxy/datatype_tabular.md]: Noted.

File: faqs/galaxy/datatype_tabular.md (hunk 18 above)
Comment (Beachyesh) [reply faqs/galaxy/datatype_tabular.md]: Done.

File: faqs/galaxy/datatype_tabular.md (hunk 16 above)
Comment (beatrizserrano) [reply faqs/galaxy/datatype_tabular.md]: @Beachyesh this commit was not done completely, do you know that you can do it directly from this comment here by clicking on "Commit suggestion" instead of editing the file?

**Comment by Beachyesh** [comment]
@shiltemann @hexylena I have made the suggested commits to these FAQs. Please let me know if there are any further suggestions for this file. Thanks!

**Comment by Beachyesh** [comment]
@beatrizserrano I apologize for my silly mistake, I just kept reading "Indentifying" as "Identifying" and there was some issue with making the suggested commit the previous time. Just a mistake on my part 🙈

File: faqs/galaxy/datatype_tabular.md [hunk 19]
```diff
@@ -0,0 +1,17 @@
...
+
+A Tabular datatype is human readable and has tabs separating data columns. Please note that tabular data is different from comma seperated data (.csv) and the common datatypes are: `.bed`, `.gtf`, `.interval`, or `.txt`.
+1. Click the pencil icon {% icon galaxy-pencil %} to reach the **_Edit Attributes_** form. 
+   1. Change the datatype (3rd tab) and save.
+   2. Label columns (1st tab) and save.
+   3. Metadata will be assigned, then the dataset can be used.
+2. If the required input is a BED or Interval datatype, adjusting (``.tab`` → ``.bed``, ``.tab`` → ``.interval``) maybe possible using a combination of **_Text Manipulation_** tools, to create a dataset that matches required specifications.
+3. Some tools require that BED format be followed, even if the datatype Interval (with less strict column ordering) is accepted on the tool form.
+   - These tools will fail, if they are run with malformed BED datasets or non-specific column assignments.
+   - **Solution** is to reorganize the data to be in BED format and rerun.
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:17]:
```suggestion
   - These tools will fail, if they are run with malformed BED datasets or non-specific column assignments.
   - **Solution**: reorganize the data to be in BED format and rerun. 
```

**Review by beatrizserrano — APPROVED** [review]

**Comment by beatrizserrano** [comment]
> @beatrizserrano I apologize for my silly mistake, I just kept reading "Indentifying" as "Identifying" and there was some issue with making the suggested commit the previous time. Just a mistake on my part see_no_evil

no worries, I've made a couple of minor suggestions and I think that we can merge very very soon, thanks @Beachyesh!

**Comment by Beachyesh** [comment]
Great! Thanks a ton! @beatrizserrano 😄

File: faqs/galaxy/datatype_tabular.md [hunk 20]
```diff
@@ -0,0 +1,18 @@
+---
+title: Identifying and formatting Tabular Datasets 
+area: Datatypes
+description: Format help for Tabular/BED/Interval Datasets
+contributors: [jennaj, beachyesh]
+layout: faq
```
Comment (beatrizserrano) [inline faqs/galaxy/datatype_tabular.md:6]: I'm not sure the order of the rows here matters, but just for consistency, could you move the contributors to the end? I think that's how it appears in the other FAQs. Thanks!

Comment (Beachyesh) [reply faqs/galaxy/datatype_tabular.md]: Done. Thanks!

**Review by beatrizserrano — APPROVED** [review]
this looks good, thanks @Beachyesh!
